### PR TITLE
Feature/app stage

### PR DESCRIPTION
**Pull request by nroode**
Added in clickable/stages in app status - though not set up as checkbox inputs so not sure if should update to this.  Fixed some formatting issues with css interactions between elements and moved the AppListing open/close state to AppListing component instead of AppList component.

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -11,7 +11,9 @@ export default function App() {
 	return (
 		<div>
 			<Header currentUser={currentUser} />
-			{auth ? <AppList /> : <Login auth={auth} setAuth={setAuth} setCurrentUser={setCurrentUser} />}
+			<div className="full-app">
+				{auth ? <AppList /> : <Login auth={auth} setAuth={setAuth} setCurrentUser={setCurrentUser} />}
+			</div>
 		</div>
```
Comment (terrancexin) [inline client/src/App.jsx:17]: what happened to the spacing? it looks like 8 tab spaces

File: client/src/App.jsx [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import React, { useState } from 'react';
 
 import AppList from './components/AppList/AppList.jsx';
-import Login from './components/Login';
-import Header from './components/Header';
+import Login from './components/Login/index.jsx';
+import Header from './components/Header/index.jsx';
```
Comment (terrancexin) [inline client/src/App.jsx:6]: redundant to add `index.jsx', it's read by default.

File: client/src/components/AppStage/AppStage.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    return (
+        <ul>
+            <li onClick={() => toggleSubmit(!submit)} className="stage">
+                <div className={submit ? "circle stage-complete" : "circle stage-incomplete"}></div>
+                <span className="status-stage"> App Submitted </span>
+            </li>
+            <li onClick={() => toggleCode(!code)} className="stage">
+                <div className={code ? "circle stage-complete" : "circle stage-incomplete"}></div>
+                <span className="status-stage"> Code Challenge </span>
+            </li>
```
Comment (terrancexin) [inline client/src/components/AppStage/AppStage.jsx:23]: `<li>`s look very similar. this is an opportunity to refactor them into one reusable component.

File: client/src/components/AppStage/AppStage.jsx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        </ul>
+
+            )
+        }
+
+        // <li onClick={() => toggleOffer(!offer)} className="stage">
+        //         <div className={offer ? "circle stage-complete" : "circle stage-incomplete"}></div>    
+        //         <input type="checkbox" name="offer" id="offer" defaultChecked={offer} />
+        //         <label for="offer" className="status-stage"> Offer </label>
```
Comment (terrancexin) [inline client/src/components/AppStage/AppStage.jsx:48]: remove comments

File: client/src/components/FullApp/FullApp.jsx [hunk 5]
```diff
@@ -1,24 +1,23 @@
 import React, { useState } from 'react';
 
+import AppStage from '../AppStage/AppStage.jsx';
+
 import './FullApp.scss';
 
 
-export default function FullApp() {
+export default function FullApp(props) {
```
Comment (terrancexin) [inline client/src/components/FullApp/FullApp.jsx:8]: you can use ES6 destructuring here `FullApp({ fullView }) {`

**Review by paul1124 — APPROVED** [review]

**Comment by nroode** [comment]
see new branch/PR refactor/appStage

**Pull request closed without merging**